examples/benchmark_stem_image.py

```diff
@@ -2,6 +2,7 @@
 from stempy import io, image
 import sys
 import time
+import numpy as np
```

Comment:
I don't think this needs to be added anymore.

Reply:
Thanks, the line has been removed.